app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comment
+  has_many :like
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, :likes_counter, comparison: { greater_than_or_equal_to: 0 }, numericality: true
+
+  def self.update_user_posts_counter(author_id)
+    u = User.find(author_id)
+    u.posts_counter += 1
+    u.save
+  end
+
```

Comment:
**[OPTIONAL]** You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_user_posts_counter` method every time a new post has been saved to database, then make the `update_user_posts_counter` method private. It will be something like this 👇 

```
 after_save :update_user_posts_counter
  ...
  private

  def self.update_user_posts_counter(author_id)
    ...
  end
```